server/graphql/v2/mutation/IndividualMutations.ts

```diff
@@ -71,7 +71,7 @@ const individualMutations = {
         }
         const validPassword = await bcrypt.compare(args.currentPassword, req.remoteUser.passwordHash);
         if (!validPassword) {
-          return new Unauthorized('Invalid current password while attempting to change password.');
+          throw new Unauthorized('Invalid current password while attempting to change password.');
```

Comment:
It was supposed to be a throw here? @znarf

Reply:
Not sure of the behavior with return, maybe it works? But anyway, `throw` looks better yes.